tests/problems/generic/test_sinkhorn_problem.py

```diff
@@ -166,3 +167,26 @@ def test_pass_arguments(self, adata_time: AnnData, args_to_check: Mapping[str, A
                 assert type(el) == type(args_to_check[arg]), arg  # noqa: E721
             else:
                 assert el == args_to_check[arg], arg
+
+    @pytest.mark.parametrize(("memory", "refresh"), [(1, 1), (5, 3), (7, 5)])
+    @pytest.mark.parametrize("recenter", [True, False])
+    def test_passing_ott_kwargs(self, adata_time: AnnData, memory: int, refresh: int, recenter: bool):
+        problem = SinkhornProblem(adata=adata_time)
+        problem = problem.prepare(
+            key="time",
+            policy="sequential",
+        )
+
+        problem = problem.solve(
```

Comment:
inner iterations = 1

Reply:
are you sure this is how to pass inner iterations? I think it's via `linear_solver_kwargs`, but not entirely sure. Please double-check.

Reply:
here it's a SinkhornProblem so inner iterations get passed just like that. In GWProblem I do pass it in ```linear_solver_kwargs={
                "inner_iterations": 1,
                "max_iterations": 1,
            }```